src/main/ipc_handler.ts

```
@@ -62,4 +71,45 @@
+      mov: "video/quicktime",
+      webm: "video/webm",
+      ogg: "video/ogg",
+      ogv: "video/ogg",
+      mpeg: "video/mpeg",
+      mp2t: "video/mp2t",
+      mpg: "video/mpeg",
+    };
+    const arrayBuffer = fileBuffer.buffer.slice(
+      fileBuffer.byteOffset,
+      fileBuffer.byteOffset + fileBuffer.byteLength,
+    );
```

ESLint

[error] Replace `⏎······fileBuffer.byteOffset,⏎······fileBuffer.byteOffset·+·fileBuffer.byteLength,⏎····` with `fileBuffer.byteOffset,·fileBuffer.byteOffset·+·fileBuffer.byteLength`

(prettier/prettier)